universal-application-tool-0.0.1/app/views/ProfileView.java

```diff
@@ -25,7 +25,7 @@ public Content render(Optional<UatProfile> maybeProfile) {
       return layout.htmlContent(
           body(
               h1(profile.getClientName()),
-              h1(profile.getId()),
+              h1(profile.getId()).withId("guest-id"),
```

Comment:
nit: this won't necessarily be a guest if the user is logged in, maybe `profile-id`?

Reply:
Great catch, thank you! Changed